net/dnscrypt-proxy/Makefile

```diff
@@ -1,5 +1,7 @@
 #
-# Copyright (C) 2009-2016 OpenWrt.org
+# Copyright (C) 2016 OpenWrt.org
+# Copyright (C) 2012-2016 Black Roland and contributors (https://github.com/black-roland/exOpenWrt/graphs/contributors)
+# Copyright (C) 2011-2012 Entware
```

Comment:
Initially this Makefile ported from [Entware project repository](https://github.com/Entware/rtndev/blob/ee16a04407e86a3c051e5f97b77bb021f83add3f/feeds/rtndev/dnscrypt-proxy/Makefile) in 2012.
Later I sent a patch to [mailing list](https://lists.openwrt.org/pipermail/openwrt-devel/2012-July/016019.html), but it was not submited. I created separate repository with this patch ([exOpenWrt](https://github.com/black-roland/exOpenWrt)).
In 2014 **damianorenfer** copied this Makefile with some modifications. As far as I remember at that time Makefile was without any copyright header at all.
Since 2014 some people contributed to exOpenWrt (procd support, updates, etc) and this changes now come in this PR.

Sorry if something is not clear. English is not my native language.